app/scripts/modules/core/src/presentation/Markdown.less

```diff
@@ -0,0 +1,9 @@
+.Markdown {
```

Comment:
Curious if the capitalization here is a convention in deck?

Reply:
Generally we apply capitalization like this when a few things are true:
1. The purpose of the class is to select all instances of a specific component type, and nothing else
2. The class sits on the root element of that component type
3. The class name is exactly identical to the name of the component type

The third point kind of goes hand-in-hand with the capitalization.

Reply:
This is a convention that has sort of organically appeared.  It's a simple way to scope CSS to a specific component.

See also: 

https://github.com/spinnaker/deck/blob/b5534656251c92180f8598cb22e8f50574e9cf1a/app/scripts/modules/core/src/presentation/forms/validation/ValidationMessage.less#L1

or in raw css:

https://github.com/spinnaker/deck/blob/23437377aace9cc6cd7fec8ecd0613b840cfe0d2/app/scripts/modules/core/src/projects/configure/FormikApplicationsPicker.css#L1

Reply:
Got it, thanks for the details.